src/backup_status.c

```diff
@@ -1507,6 +1507,9 @@ get_object_count(aerospike *as, const char *namespace, as_vector* set_list,
 		*obj_count += count;
 	}
 
+	// if effective replication factor is 0 set it to 1 to prevent division by 0
+	effective_repl_factor = effective_repl_factor == 0 ? 1 : effective_repl_factor;
```

Comment:
What's the goal of changing it to 1?  I would think a warning like the one you have above would be a better approach. Odds are the estimate is going to be wrong if object count != 0 since the effective replication factor was likely > 1 at one point

Reply:
It's to prevent a divide by 0 error. This value is divided into obj_count later.

A warning is a good idea. I'll add one.